### ISSUE TITLE
Vercel Clone Not Working

### ISSUE THREAD

**Issue by crosleythomas** [Nutlope/alt-text-generator#1]
I've tried cloning/deploying this project directly on Vercel, but it seems to be broken

![image](https://github.com/Nutlope/alt-text-generator/assets/5376546/6946ea87-339f-4610-a0ab-4a9614bb9c7a)

The logs only say `500 Unknown application error occurred Runtime.Unknown`

I added my API key and didn't see any other setup instructions.

**Comment by eliiiii** [comment]
Also having this issue, doesn't build locally with `vercel dev` either. Trying to use replicate in any other vercel project using flask also fails both locally and on vercel servers. Wonder if anyone has found a solution? Merely adding an `import replicate` statement in app.py leads to a 502: bad gateway (NO_RESPONSE_FROM_FUNCTION).

Using `vercel dev`, logs show 
```
TypeError: Retry.__init__() got an unexpected keyword argument 'method_whitelist'
TypeError: Object prototype may only be an Object or null: undefined
    at Function.setPrototypeOf (<anonymous>)
    at new LambdaError (/usr/local/lib/node_modules/vercel/dist/index.js:12845:16)
    at Lambda.<anonymous> (/usr/local/lib/node_modules/vercel/dist/index.js:12978:27)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/vercel/dist/index.js:12892:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```
along with another dozen lines of python traceback.

Happy to provide more information (I'm trying to get replicate running with flask on vercel for a different project)

--
_MacBook Pro M1 Pro (Late 2021), MacOS 13.2, Vercel CLI 31.4.0, Python 3.11.2_

**Comment by mattt** [comment]
@crosleythomas @eliiiii @Nutlope 👋 Hey there. Thanks for reporting this and apologies for the inconvenience. I believe this is the same issue as https://github.com/replicate/replicate-python/issues/109, which has to do with an API change in urllib3.

Upstream in Replicate's Python client, I'll look into updating the version requirements for urllib3 to get pip to resolve to a compatible version. In the meantime, you might be able to work around this by pinning and upgrading to the latest version manually.

**Comment by eliiiii** [comment]
@mattt thanks so much! Appreciate your reply, excited to get it working :)

**Comment by mattt** [comment]
Just saw a notification that this was closed. Sorry for not following up. This was fixed by https://github.com/replicate/replicate-python/pull/147, which is available in `replicate>=0.14.0`.
